packages/web/components/cards/pool-card.tsx

```diff
@@ -0,0 +1,197 @@
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
+                alt="no token icon"
+                width={40}
+                height={40}
+              />
+            </div>
+          )}
+        </div>
+        <div className="flex flex-col">
+          <h5>
+            {pool.poolAssets.length >= 3
+              ? `${pool.poolAssets.length} Token Pool`
+              : pool.poolAssets
+                  .map((asset) => asset.amount.currency.coinDenom)
+                  .join(" / ")}
+          </h5>
+          <div className="subtitle2 text-white-mid">{`Pool #${pool.id}`}</div>
+        </div>
+      </div>
+      <div className="mt-5 mb-3 w-full bg-secondary-200 h-[1px]" />
+      <div className="flex flex-wrap gap-x-8">
+        <div className="flex flex-col">
+          <div className="subtitle2 text-white-disabled">APR</div>
+          {queryOsmosis.queryIncentivizedPools.isAprFetching ? (
+            <div className="relative overflow-hidden rounded-sm w-[3.75rem] h-4 bg-white-faint mt-[0.4375rem]">
+              <div className="absolute left-0 -translate-x-[calc(-150%)] h-full w-1/2 bg-loading-bar animate-loading" />
+            </div>
+          ) : (
+            <div className="mt-0.5 subtitle1 text-white-high">{`${apr.toString()}%`}</div>
+          )}
+        </div>
+        <div className="flex flex-col">
+          <div className="subtitle2 text-white-disabled">Pool Liquidity</div>
+          {poolLiquidity.toDec().isZero() ? (
+            <div className="relative overflow-hidden rounded-sm w-[6.5rem] h-4 bg-white-faint mt-[0.4375rem]">
+              <div className="absolute left-0 -translate-x-[calc(-150%)] h-full w-1/2 bg-loading-bar animate-loading" />
+            </div>
+          ) : (
+            <div className="mt-0.5 subtitle1 text-white-high">
+              {poolLiquidity.toString()}
+            </div>
+          )}
+        </div>
+        <div className="flex flex-col">
+          <div className="subtitle2 text-white-disabled">
+            {isMyPool ? "Bonded" : "Fees"}
+          </div>
+          {isMyPool ? (
+            poolLiquidity.toDec().isZero() ? (
+              <div className="relative overflow-hidden rounded-sm w-[3.75rem] h-4 bg-white-faint mt-[0.4375rem]">
+                <div className="absolute left-0 -translate-x-[calc(-150%)] h-full w-1/2 bg-loading-bar animate-loading" />
+              </div>
+            ) : (
+              <div className="mt-0.5 subtitle1 text-white-high">
+                {bonded.toString()}
+              </div>
+            )
+          ) : (
+            <div className="mt-0.5 subtitle1 text-white-high">
+              {pool.swapFee.toString()}
```

Comment:
I know we'll want to reuse this great pool card view logic in the assets page, but with different data metrics. Consider pulling out all the data fetching done here and instead accept an array of at most 3 simple value objects for the bottom half. See `/components/overview/**`.